### ISSUE TITLE
Faster ways to add faves

### ISSUE THREAD

**Issue by tportersplash** [leftstick/vscode-favorites#64]
Awesome extension! thank you!

Is it possible to get a shortcut key working with this?...or if not possible, at least an option in the context menu from the opened files tabs?

![image](https://user-images.githubusercontent.com/62571730/169189812-50225ea1-0a02-41c0-8344-3e90bd9fe6df.png)

**Comment by leftstick** [comment]
@tportersplash yes, it is possible. Do you have any suggested combinations?

**Comment by tportersplash** [comment]
> @tportersplash yes, it is possible. Do you have any suggested combinations?

Good question :) 
Maybe `CTRL+SHIFT+F` ??

**Comment by leftstick** [comment]
@tportersplash  `ctrl+shift+f` has been taken by "search: find in files" function.
The reason I didn't put a shortcut is most combinations I preferred have been taken by other functions. It is so sad

**Comment by tportersplash** [comment]
True, that sucks....maybe just add an option in this context menu then??

![image](https://user-images.githubusercontent.com/62571730/169545705-c008a97c-6c95-4f6c-85fc-e50495c943bb.png)

**Comment by tportersplash** [comment]
or what about `ctrl+option+shift+F` ?

**Comment by leftstick** [comment]
> or what about `ctrl+option+command+F` ?

We should consider both windows and osx. `ctrl+option+command+F` makes sense only with windows since the equivalent key for `command` in windows might be `win`, which is not a common key for shortcut case

**Comment by tportersplash** [comment]
OK, seems we have to scrap the shortcut key idea for now then. 
What about the other suggestion above?

**Comment by leftstick** [comment]
Thank you for your suggestion. You can assign a keyboard shortcut to the "favorites.addToFavorites" command through VS Code's keyboard shortcut settings. This will allow you to quickly add files to your favorites without using the context menu.
